spec/formatting.md

```diff
@@ -222,6 +222,74 @@ the following steps are taken:
 
    The form that resolved _operand_ and _option_ values take is implementation-defined.
 
+   A _local-declaration_ binds the resolved value of an _expression_
+   to a _variable_.
+   Thus, the output of one _function_ is potentially the _operand_
+   of another _function_,
+   or the value of one of the _options_ for another function.
+   For example, in
+   ```
+   .input {$n :number minIntegerDigits=3}
+   .local $n1 = {$n :number maxFractionDigits=3}
+   ```
+   the output of the first call to `:number`
+   is the input of the second call to `:number`.
+
+   Implementations that provide a means for defining custom functions
+   SHOULD provide a means for function implementations
+   to return values that contain enough information
+   (e.g. the resolved _operand_ and _option_ values
+   that the function was called with)
+   to be used as arguments to subsequent calls
+   to the function implementations.
+   For example, a MessageFormat implementation in a typed programming language
+   MAY define an interface for custom function implementations.
+   Such an interface SHOULD define an implementation-specific
+   argument type `T` and return type `U`
+   for implementations of formatting functions
+   such that `U` can be coerced to `T` without loss of information.
+   The type `U`
+   (or a type that `U` can be coerced to without loss of information)
+   SHOULD also be the input type of implementations of
+   custom selector functions.
+   Implementations of specific functions MAY
+   signal errors if supplied an _operand_ that
+   does not make sense for the particular function
+   being implemented.
+
+> [!NOTE]
+> The behavior of the previous example is
+> implementation-dependent. Supposing that
+> the external input variable `n` is bound to the string `"1"`,
+> and that the implementation formats to a string,
+> the formatted result of the following message:
+>
+> ```
+> .input {$n :number minIntegerDigits=3}
+> .local {$n1 :number maxFractionDigits=3}
+> {{$n1}}
```

Comment:
This is not valid syntax. Did you mean the below?
```suggestion
> .input {$n :number minIntegerDigits=3}
> .local $n1 = {$n :number maxFractionDigits=3}
> {{$n1}}
```

Reply:
Thanks, I forgot to correct this error in both places